src/observables/distfit.py

```diff
@@ -123,21 +128,20 @@ def get_continuous_dist(var, var_val, cont_dists, alt_name=None):
     cont_dict = defaultdict(dict)
     # Use the Fitter library to fit distributions
     # to the data
-    fitter_object = Fitter(data=var_val,
-                           distributions=cont_dists,
-                           timeout=60,
-                           verbose=True)
+    fitter_object = Fitter(
+        data=var_val, distributions=cont_dists, timeout=60, verbose=True
```

Comment:
The `verbose=True` specification gives me the following error when I try to run your public notebook on my computer.
<img width="1140" alt="Screen Shot 2020-11-07 at 3 31 38 PM" src="https://user-images.githubusercontent.com/5533699/98453558-5fc67600-210f-11eb-8eae-85ec5f7aca21.png">

I remember you said you had to do something to fitter to get this to work? Is there anyway we can do without choices like that? It makes it impossible for our work to be reproduced.

Reply:
Just fixed it in the new version. It seems like a newer version of the fitter package gets rid of this verbose argument.